src/pages/Main/components/Sidebar/services/group-services.js

```diff
@@ -0,0 +1,11 @@
+export const searchGroup = (target, groups) => {
+    if (target !== '') {
+        const filteredGroups = groups.filter((group) => {
```

Comment:
you can just return here and make ? : operator

Reply:
and you can just check !target, because '' == false

Reply:
return !target ? groups.filter((group) => {}) : groups;

if(target) {
return groups; }

return groups.filter((group) => {}) ;